test/integration/admin_edits_item_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class AdminEditsItemTest < ActionDispatch::IntegrationTest
+
+  test 'admin can edit an item' do
+    # Background: an existing item
+    create_category_and_items(1)
+    logged_in_admin
+    #       As an admin
+    visit admin_items_path
+    #       When I visit "admin/items"
+    item1 = Item.find_by(title: 'pour over0')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.